ckan/views/group.py

```diff
@@ -604,7 +603,6 @@ def member_dump(id: str, group_type: str, is_organization: bool):
         ])
 
     output_stream = StringIO()
```

Comment:
Let's keep the BOM (just fix the type) so that non-ASCII characters are rendered properly in Excel

```suggestion
    output_stream = StringIO()
    output_stream.write('\N{BOM}')  # for Excel handling of non-ASCII
```